### PR TITLE
Added new courses , a MOOC site, and a programming playground

### PR DESCRIPTION
**Pull request by Divya4879**
## What does this PR do?
Add resource(s) -
1) Added a Mooc resource- site link
2) Added 1 course on these topics - > Linux
                                                          > Machine Learning
                                                          > Django
                                                          > Web Development
                                                          > Web3
3) Added a playground (for multiple clouds) site

                              
## For resources
# Courses- Added 5 courses and a MOOC site
All the courses are on YouTube and hence free
MOOC courses are all free

#Playground - Added a site for free playgrounds on multiple clouds

### Why is this valuable (or not)?
All the courses, the MOOC are invaluable as they have great content, all have many thankful comments and reviews, are completely free

the playground is great for free practice

### How do we know it's really free?
All the 5 courses are on YouTube, and hence completely free
The MOOC site has all free courses
The playground is free to use

### For book lists, is it a book? For course lists, is it a course? etc.
Added 5 courses and a MOOC site link in courses/free-courses-en.md
Added a playground in more/free-programming-playgrounds.md

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Divya4879** [comment]
@eshellman Sir kindly review my PR, suggest the changes if needed, and merge my PR. Thank you for your time.

**Comment by Divya4879** [comment]
@eshellman Also sir, will it count as a non-coding contribution in hacktoberfest?

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1384,11 +1387,13 @@
 
 * [Django 3.0 Crash Course Tutorials \| Customer Management App](https://www.youtube.com/playlist?list=PL-51WBLyFTg2vW-_6XBoUpE7vpmoR3ztO) - Dennis Ivy
 * [Django for Everybody](https://www.dj4e.com) - Charles R. Severence
+* [Django For Everybody - Full Python University Course](https://www.youtube.com/watch?v=iwRneX7GIGI) - freeCodeCamp.org
 * [Django Tutorial for Beginners](https://www.youtube.com/playlist?list=PLsyeobzWxl7r2ukVgTqIQcl-1T0C2mzau) - Navin Reddy
 * [Django tutorial for Beginners](https://www.youtube.com/playlist?list=PLK8cqdr55Tsv-D2HMdrnD32oOVBNvmxjr) - ProgrammingWithHarry
 * [Django Tutorial for Beginners (2021)](https://www.youtube.com/watch?v=rHux0gMZ3Eg) - Moshfegh Hamedani (Programming with Mosh)
 * [Django Tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTtoQCKZ03TU5fNfx2UY6U4p) - Corey Schafer
 * [Django Tutorials for Beginners](https://www.youtube.com/playlist?list=PLK8cqdr55Tsv-D2HMdrnD32oOVBNvmxjr) - Programming With Harry(English)
+
```
Comment (eshellman) [inline courses/free-courses-en.md:1396]: typo

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1384,11 +1387,13 @@
 
 * [Django 3.0 Crash Course Tutorials \| Customer Management App](https://www.youtube.com/playlist?list=PL-51WBLyFTg2vW-_6XBoUpE7vpmoR3ztO) - Dennis Ivy
 * [Django for Everybody](https://www.dj4e.com) - Charles R. Severence
+* [Django For Everybody - Full Python University Course](https://www.youtube.com/watch?v=iwRneX7GIGI) - freeCodeCamp.org
```
Comment (eshellman) [inline courses/free-courses-en.md:1390]: video does not correspond to course

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [inline courses/free-courses-en.md:1396]: typo

File: more/free-programming-cheatsheets.md [hunk 3]
```diff
@@ -379,6 +379,7 @@
 #### Jupyter
 
 * [Jupyter Cheatsheet](https://www.datacamp.com/cheat-sheet/jupyter-notebook-cheat-sheet) - Data Camp (HTML site with PDF links)
+* [Jupyter Notebook Cheat Sheets](https://ugoproto.github.io/ugo_py_doc/jn_cs/) - ugo_py_doc (HTML site with PDF links)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:382]: cheatsheets copied from anaconda site? Use original source.

File: more/free-programming-playgrounds.md [hunk 4]
```diff
@@ -385,3 +387,8 @@
 ### V
 
 * [V Playground](https://play.vlang.io) - vlang.io
+
+
+### Multiple Clouds
+
+* [KodeKloud](https://kodekloud.com/playgrounds/)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:394]: our list of playgrounds does not link to lists of playgrounds

File: more/free-programming-playgrounds.md [hunk 5]
```diff
@@ -423,8 +426,10 @@
 * [V Playground](https://play.vlang.io) - vlang.io
 
 
+
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:429]:
```suggestion
```

File: more/free-programming-playgrounds.md [hunk 6]
```diff
@@ -27,6 +27,7 @@
     * [AngularJS](#angularjs)
 * [Kotlin](#kotlin)
 * [Kubernetes](#kubernetes)
+* [Multiple Clouds](#multiple-clouds)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:30]:
```suggestion
```

File: courses/free-courses-en.md (hunk 2 above)
Comment (eshellman) [reply courses/free-courses-en.md:1492]:
```suggestion
```

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-en.md:1497]:
```suggestion
```

File: more/free-programming-cheatsheets.md (hunk 3 above)
Comment (eshellman) [reply more/free-programming-cheatsheets.md:419]:
```suggestion
```

File: more/free-programming-playgrounds.md [hunk 7]
```diff
@@ -261,6 +261,7 @@
 * [CodeChef](https://www.codechef.com/ide)
 * [GeeksforGeeks](https://ide.geeksforgeeks.org)
 * [Ideone](https://ideone.com)
+* [KodeKloud](https://kodekloud.com/playgrounds/)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:264]: duplicate
```suggestion
```

**Pull request merged**